tests/git-auto-commit.bats

```diff
@@ -87,8 +87,8 @@ git_auto_commit() {
     assert_success
 
     assert_line "INPUT_REPOSITORY value: ${INPUT_REPOSITORY}"
-    assert_line "::set-output name=changes_detected::true"
-    assert_line -e "::set-output name=commit_hash::[0-9a-f]{40}$"
+    assert_line '"changes_detected=true" >> $GITHUB_OUTPUT'
```

Comment:
Oh wait. Yeah this probably isn’t right is it? Now that I read this I realize this is probably directly comparing the contents of stdout line by line. And none of this is gonna be there any more. So probably need a way to validate the `$GITHUB_OUTPUT` contents. :-\